packages/ra-core/src/controller/useListController.spec.tsx

```diff
@@ -88,37 +88,6 @@ describe('useListController', () => {
             });
         });
 
-        it('should not call filtering function if filters are unchanged', () => {
```

Comment:
why did you remove that test?

Reply:
Because I removed the equality check in the setFilters function. Could not make it work otherwise

Reply:
What's the impact?

Reply:
I have not seen any but as we already know, a global performance check will be needed for the 3.0